Makefile

```diff
@@ -462,21 +463,22 @@ test-and-cover:
 		tail -n +2 profile_tmp.cov >> profile.cov; \
 	done
 
-android-lib: docker-golang-android genconfig
+android-lib-dev: docker-mobile
```

Comment:
This `android-lib-dev` target looks a lot like `android-lib`, the only difference I can tell is this line:

```
if [ -d "$(FLASHLIGHT_ANDROID_TESTER_DIR)" ]; then \
```

compared to

```
if [ -d "$(FIRETWEET_DIR)" ]; then \
```

I think we can just leave one `android-lib` target and use environment variables, we'll have to do something like this:

```
FIRETWEET_DIR=../flashlight-android-tester make android-lib
```

for the current `android-lib-dev` or:

```
make android-lib
```

for the default `android-lib` case.